app/autoload.php

```diff
@@ -6,7 +6,7 @@
 /**
  * @var ClassLoader $loader
  */
-$loader = require __DIR__.'/../vendor/autoload.php';
```

Comment:
Yes indeed, as the vendors are not anymore on the host, but only in the container. These commits serve as a test for problems with Docker on Mac, so you may not use it.
If you keep the vendors in the container (like in this PR), you may temporarily change the `composer.json` file to make vendors get installed in the standard dir as well, so that your IDE can see the files.